infrastructure/models.py

```diff
@@ -50,22 +66,40 @@ def __str__(self):
 
 
 class ProjectStatus:
+    NULL = 0
     ANNOUNCED = 1
     PREPATORY = 2
     STARTED = 3
     UNDER_CONSTRUCTION = 4
     COMPLETED = 5
     CANCELLED = 6
+    ACTIVE = 7
```

Comment:
I'm not positive about this, but I would recommend putting these plant-specific statuses (ACTIVE, PARTIALLY_ACTIVE, INACTIVE) into a `class PpwerPlantStatus`, since they're only for the `PowerPlant` model